libenv/sysinfo.c

```diff
@@ -3335,6 +3335,80 @@ void GetDefVars(EvalContext *ctx)
         "jq --compact-output --monochrome-output --ascii-output --unbuffered --sort-keys",
         CF_DATA_TYPE_STRING, "invocation,source=agent,command_name=jq");
 }
+
+/*******************************************************************/
+
+static void SysOSNameHuman(EvalContext *ctx)
+{
+    assert(ctx != NULL);
+    const char *const lval = "os_name_human";
+    const char *const tags = "source=agent";
+
+    /**
+     * Order is important.
+     * E.g.
+     * |    A    |    B    | A -> B |
+     * |---------|---------|--------|
+     * | ubuntu  | debian  | true   |
+     * | debian  | ubuntu  | false  |
```

Comment:
```suggestion
     * Order is important. The first if is more specific and wins,
     * for example, prefer Ubuntu over Debian.
```

This comment makes me more confused, I would go for a simpler sentence and no table.

Reply:
Yes, your totally right. Guess I got a little inspired by the illustration you showed me in the docs on files promise-type.